### DISCUSSION TITLE
UCP bypass

### DISCUSSION THREAD

**Discussion by DevonH99** [OwlGamingCommunity/MTA#65] [Support] [answered]
Looking to play around on localhost made account but I get the error "You're not allowed to connect to server from that PC, check UCP." anyone know a bypass for this?

**Comment by saviorsoldier** [comment]
If I remember correctly you need to add your MTA serial to one of the database table that contains allowed serial of the users. If there's not table related to that check the accounts table.

**Comment by skoozeblood** [comment]
Also having this issue, which table is it?

**Comment by starinjs** [comment, marked as answer]
serialwhitelist table for anyone thats wondering.
